src/pages/unsubscribe.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import styled from 'styled-components'
+import { validate } from 'email-validator'
+
+import Section, { TopSection } from '../components/layout/Section'
+import { InputButton } from '../components/buttons/Button'
+import DefaultInput, { ErrorMessage } from '../components/form/Input'
+import Grid, { Col, Row } from '../components/layout/Grid'
+import { H2, H3, P } from '../components/text'
+import Header from '../components/layout/Header'
+import { WHITE, FONT_FAMILY } from '../config/styles'
+
+import { Label as DefaultLabel } from '../components/text'
+
+const Input = styled(DefaultInput)`
+  background-color: ${WHITE};
+`
+
+const Label = styled(DefaultLabel)``
+
+const ThanksTitle = styled(H3)`
+  margin: 1em 0;
+` //TODO: animate this later
+
+class Unsubscribe extends React.Component {
+  state = {
+    email: '',
+    formSubmited: false,
+    emailValid: false,
+  }
+
+  handleFormSubmit = e => {
+    e.preventDefault()
+    this.setState({ formSubmited: true })
+    this.triggerUnsubscribe()
+  }
+
+  triggerUnsubscribe = () => {
+    fetch(
+      `https://us-central1-reactjsacademy-react.cloudfunctions.net/unsubscribe`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          email: this.state.email,
+        }),
+      }
+    )
+  }
+
+  handleEmailChange = e => {
+    this.setState({ email: e.target.value }, this.validateEmail)
+  }
+
+  validateEmail = () => {
```

Comment:
@ric9176 why don't we use final-form to keep the state of the input and validate it? it makes the code simpler and we are already using final-form in this project

Reply:
good point. I'll make that refactor, me and @horacioh did this quick version last night

Reply:
are you going to do this changes @ric9176 ?

Reply:
@horacioh I've done and pushed here the first change already if you want to do this one (Final form)

Reply:
cool!